src/Symfony/Component/Validator/Tests/Constraints/ExpressionValidatorTest.php

```diff
@@ -304,4 +306,24 @@ public function testViolationOnPass()
             ->setCode(Expression::EXPRESSION_FAILED_ERROR)
             ->assertRaised();
     }
+
+    public function testIsValidExpression(): void
```

Comment:
I would add another test case with a value that is not valid

Reply:
added